assets/stylesheets/admin-pages.scss

```diff
@@ -383,3 +383,19 @@ table#users {
     padding-right: 1rem;
     font-weight: 500;
 }
+
+// custom select (used on needles page)
+.custom_select {
+    display: inline-block;
+    width: 18%;
```

Comment:
This seems to result in much too small controls where you can't even read the options unless the window is very wide.

Reply:
I could change this from `18%;` to a number width, such as`100px;`. But this will not become wider when the window is wider. Is it ok ?

Reply:
It looks fine to me without the `width` set. If you can give it a width that ensures all labels are visible that's also fine by me.

Reply:
I gave them a width to ensure all labels will be shown in one line when window is wide enough.